### PR TITLE
feat: Sharpen an image

### PR DESCRIPTION
**Pull request by viveknathani**
There are two ways to do this. (i) unsharp masking and (ii) convolution
Currently, convoultion is implemented since (ii) is faster than (i). In
future, (i) can be implemented too to make the software feature-rich.

### CODE REVIEW

File: src/secur3dit/filters/Filters.java [hunk 1]
```diff
@@ -502,4 +502,58 @@ public static BufferedImage pixelate(BufferedImage image, int pixelWidth) {
...
+    /**
+     * Takes an image and returns a sharpened version of it
+     * with given intensity. The intensity parameter is used to produce
+     * a 3x3 kernel which will be convolved with the image. The process is 
+     * similar to blurring an image using box blur just once. Since the kernel 
+     * is just 3x3, the total steps for convolutionResult calculation in each 
+     * iteration will always be 9. Hence, this kernel is not made separable and 
+     * works in O(height * width), similar to box blur with a separable kernel 
+     * and varying kernelRadius. The intensity can theoretically range from 
+     * [0, infinity]. However, in practice, after a threshold, the higher values
```
Comment (priyanshul-govil) [inline src/secur3dit/filters/Filters.java]: should be [0, infinity)

**Review by priyanshul-govil — CHANGES_REQUESTED** [review]
need change in the specified range of values in the comments.

**Comment by viveknathani** [comment]
fixed

**Review by priyanshul-govil — APPROVED** [review]
seems good.

**Pull request merged**